### DISCUSSION TITLE
The Moddable SDK now requires ESP-IDF v4.4.3 for ESP32 builds

### DISCUSSION THREAD

**Discussion by andycarle** [Moddable-OpenSource/moddable#930] [Announcements]
Today's Moddable SDK update brings our supported ESP-IDF version up to v4.4.2. If you're an ESP32 user, after pulling from the Moddable SDK repo, you'll want to:

 - clear out any existing `bin/esp32` and `tmp/esp32` build output
 - update your ESP-IDF to the `v4.4.2` tag
 - re-run the `install.sh` (macOS & Linux) / `install.bat` (Windows) script in the updated ESP-IDF directory
 - launch a new terminal / command prompt to use the updated environment

Much more detailed instructions are in our [Using the Moddable SDK with ESP32](https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/devices/esp32.md) documentation.

For my fellow Windows users, I also strongly recommend re-running the ESP-IDF Windows Installer to get updated Windows Security exclusions installed. I forgot to do that myself this afternoon and was very confused about why my ESP32 builds were suddenly taking dozens of minutes...

Feel free to post a reply here or ping us on [Gitter](https://gitter.im/embedded-javascript/moddable) if you have any questions or run into any trouble

&nbsp;- Andy

**Comment by mkellner** [comment]
Edit to update to v4.4.3 for Moddable SDK 3.5.0
